From 2b0e26615026ee9df3e234a514275b1a6d2ef10e Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:20 -0400
Subject: [tools] lib/traceevent: Use global QUIET_CLEAN build output

Message-id: <1408953210-26343-193-git-send-email-jolsa@redhat.com>
Patchwork-id: 88868
O-Subject: [PATCH RHEL7.1 BZ1133083 192/322] tools lib traceevent: Use global QUIET_CLEAN build output
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 4a953c716af285c94432e2d303ca5ee8f66129c0
Author: Jiri Olsa <jolsa@redhat.com>
Date: Thu Dec 19 14:42:06 2013 +0100

description
===========
Using global QUIET_CLEAN build output variable and so we could have only
single clean message:

  CLEAN libtraceevent
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/lib/traceevent/Makefile b/tools/lib/traceevent/Makefile
index 8ea4368..3496c9c 100644
--- a/tools/lib/traceevent/Makefile
+++ b/tools/lib/traceevent/Makefile
@@ -328,8 +328,9 @@ install_plugins: $(PLUGINS)
 install: install_lib
 
 clean:
- $(RM) *.o *~ $(TARGETS) *.a *.so $(VERSION_FILES) .*.d
- $(RM) TRACEEVENT-CFLAGS tags TAGS
+ $(call QUIET_CLEAN, libtraceevent) \
+  $(RM) *.o *~ $(TARGETS) *.a *.so $(VERSION_FILES) .*.d \
+  $(RM) TRACEEVENT-CFLAGS tags TAGS
 
 endif # skip-makefile
 
-- 
1.7.1